### DISCUSSION TITLE
Thank you! ❤️

### DISCUSSION THREAD

**Discussion by Moonbase59** [AzuraCast/AzuraCast#7288] [General]
From just an evaluation of AzuraCast’s usability for a purpose, to now using it and developing the [Autocue](https://github.com/Moonbase59/autocue) feature and other [useful](https://github.com/Moonbase59?tab=repositories) [things](https://gist.github.com/Moonbase59). This has become a full-time job taking much more than normal working hours each day.

Not to mention @BusterNeece, who has to struggle with an _immense_ workload, keeping things up-to-date, developing new features and taking care of all the many issues that arise.

_Did you know most software projects like these are essentially one-person shows?_

**And we _love_ what we’re doing. Or we wouldn’t do it. Know how that is, don’t you?**

## I just feel it’s time again to say a heartfelt "Thank you!"

For the great software. For the wonderful collaboration, both in front of and behind the scenes. For help. For new friendships.

**We do it for free.** Because we love what we do. And we believe in the idea of Free and Open Source Software (FOSS). We believe that knowledge should be shared, so that _everyone_ has access.

**But FOSS can only survive if the developers survive.** Sometimes not seeing a single donation for _years_, and still continuing working on it many hours a day can bring severe frustration, and sometimes an abrupt end. All while seeing that others take your work and make money with it.

## So please, if you can anyhow afford it, or make money with it, give back!

It’s as easy as hitting the ![Sponsor](https://github.com/AzuraCast/AzuraCast/assets/3706922/7116d88b-b1a7-4688-8d57-43961195563e) or ![Auswahl_468](https://github.com/AzuraCast/AzuraCast/assets/3706922/0cebbe2e-e147-49b5-826b-a4bd1ab9156c) buttons _(No, not these!)_ **in the [AzuraCast](https://github.com/AzuraCast/AzuraCast) and [Autocue](https://github.com/Moonbase59/autocue) repositories** and donate a _one-time_, _monthly_ or _yearly_ amount!

**Comment by marekdziem** [comment]
HEAR HEAR!
:)

**Comment by bombelman** [comment]
I don't see the "donate" button, I do see a "buy me coffee" button for a "Silver Eagle". Is that Buster Neece also?

**Reply by Moonbase59** [reply]
Yes, Silver Eagle is Buster Neece.

**Sponsoring/Donation options on GitHub can be found…**

### Near the _top_ of a repository (Sponsor button):

![Screenshot 2024-07-05 at 14-32-21 AzuraCast_AzuraCast A](https://github.com/AzuraCast/AzuraCast/assets/3706922/b240e751-9b32-4997-8375-bc99cc363534)

### In the right sidebar of a repository (Sponsoring options):

![Screenshot 2024-07-05 at 14-34-50 AzuraCast_AzuraCast A](https://github.com/AzuraCast/AzuraCast/assets/3706922/f5e57b44-b203-4d59-bf4b-855cc5028b8e)

### In the README file (Donate button):

![Screenshot 2024-07-05 at 14-37-20 Moonbase59_autocue On](https://github.com/AzuraCast/AzuraCast/assets/3706922/8bc39d2e-315f-4b73-a7c4-79902c4a7722)
